### ISSUE TITLE
sync only active tab

### ISSUE THREAD

**Issue by backpacker69** [micimize/chizukeki#20]
when you switch to Assets, chizukeki still queries explorer for new transactions

**Comment by micimize** [comment]
I don't think this is necessary, now that syncing has been optimized to not refetch transactions with #14. Seeing a transaction visible on the explorer might eventually be a good way to double check papi, and the wallet data is still relevant to assets, like if they make a transaction elsewhere with the same wallet utxos will change

**Comment by micimize** [comment]
also, the user can manually disable syncing already if they really want to
